linopy/solvers.py

```diff
@@ -845,6 +863,45 @@ def get_solver_solution() -> Solution:
     return Result(status, solution, m)
 
 
+mosek_bas_re = re.compile(r" (XL|XU)\s+([^ \t]+)\s+([^ \t]+)| (LL|UL|BS)\s+([^ \t]+)")
+
+
+def run_mosek_remote(
```

Comment:
I would honestly avoid this function. This contains one parameter change and does not represent a dedicated solver. I would instead set the parameter as a default in the standard `run_mosek` and let people change it to their needs.

Reply:
I agree - this is a hack.. The problem is that I don't like setting remote optimization as the default, but I need to use it to make the tests run.

Reply:
I'll set the default to remote optimization, but I would like that to be a temporary fix only.

Reply:
I have removed the `mosek_remote`. This means that tests won't run in the CI, so I'd like to find a solution around that. What do you do for Gurobi? Does the CI image contain a license?